docroot/themes/humsci/su_humsci_theme/patterns/molecules/generic-three-column/generic-three-column.html.twig

```diff
@@ -0,0 +1,41 @@
+{% set attributes = attributes.addClass('generic-three-column', variant|clean_class) %}
+
+{% set leftAttributes = create_attribute({class:'left-region'}) %}
+{% set middleAttributes = create_attribute({class:'middle-region'}) %}
+{% set rightAttributes = create_attribute({class:'right-region'}) %}
+
+{% if 'static' in variant %}
+  {% set static = true %}
+{% endif %}
+
+{% if '25_50_25' in variant %}
+  {% set middleAttributes = middleAttributes.addClass('larger') %}
+{% elseif '50_25_25' in variant %}
+  {% set leftAttributes = leftAttributes.addClass('larger') %}
+{% elseif '25_25_50' in variant %}
+  {% set rightAttributes = rightAttributes.addClass('larger') %}
+{% endif %}
+
+{% spaceless %}
+  <div{{ attributes }}>
+
+    {%- if left|has_markup or static -%}
```

Comment:
I don't think you need the `-` here when you wrap the whole thing in `{% spaceless %}`.